projects/vcx.md

```diff
@@ -0,0 +1,97 @@
+
+Given it's long history, VCX had previously grown in legacy code, however recent efforts have pushed VCX inline with modern DI. These efforts include complete replacement of Indy SDK components with indy-vdr, askar & anoncreds-rs, and moving away from unqualified DIDs/resources with alternatives such as Cheqd Anoncred methods.
+
+VCX aims to continue this trajectory. Some current future plans include:
+* VCX Framework (in development) - A credo-like framework solution for Rust-based Aries/DI consumers.
+* Mobile bindings - Provide Android/iOS mobile bindings to the library, allowing *native* (Kotlin/Swift) mobile applications to utilize VCX functionality.
+* More DID methods - continue growing the suite of DID method options, with modern methods such as did:webvh
+* DIDComm v2 support
+
+# Alignment with the OpenWallet Foundation Mission
+
+VCX supports the OWF's goals by providing an extensive suite of tools to assist in the creation of secure & interopable DI applications.
+* Secure Data Management: Whenever VCX manages data for the consumer, security is always the first consideration. VCX achieves secure data management via utilization of OWF's Askar project for cryptography and encrypted data storage.
+* Interoperability: As mentioned, VCX comes from an Aries ecosystem, where interoperability is at the forefront. VCX holds itself accountable to this by participating in efforts such as the OWF OWL test harness.
+* Open Source Development: VCX has always been community driven, and encourages participation with efforts such as fortnightly community calls and previous participation in the Linux Foundation Mentorship programs.
+* Support for Decentralized Identity: VCX's purpose is to provide tools that make it easier for consumers to develop DI applications. VCX has been proven to succeed in this, with testimonies in the industry (such as from [Anonyome Labs](https://anonyome.com/resources/blog/aries-vcx-anonyomes-commitment-to-decentralized-identity/)).
+
+# Code of Conduct
+
+VCX currently follows the Hyperledger Code of Conduct, which is what the OWF code of conduct is based on.
+
+# TAC Sponsor
+
+Stephen Curran
+Second Sponser - TBD
+
+# Project License
+
+Apache 2.0
+
+# Source Control
+
+Project already exists on Github: https://github.com/hyperledger/aries-vcx .
+
+Ideally the new slug would be `vcx`. Dropping the Aries to reflect the capabilities beyond Aries protocols.
+
+# Issue Tracker
+
+https://github.com/hyperledger/aries-vcx/issues
+
+# External Dependencies
+
+The VCX dependency list is maintained in the [source repository](https://github.com/hyperledger/aries-vcx/blob/main/Cargo.toml). All are open source and we believe (but to be verified) that the majority use the Apache 2 License.
```

Comment:
I suggest also to check if any of  these dependencies brings a license (open source or not) that collides with the permissive nature of Apache.

Reply:
i've ran a simple check with `cargo license`:
* majority are apache 2.0,
* others include: bsd-3-clause, ISC, MIT, ZLib
* MPL-2.0 is used by uniffi-rs deps, however i believe it's not a problem since we are not modifying the uniffi code at all, simply consuming it
* Unicode-3.0 is used by some icu_ deps, however we are not re-distributing or export their unicode functionality, so i don't believe further license restrictions apply
* finally [ring](https://github.com/briansmith/ring/blob/main/LICENSE) has a custom license based on boringssl/openssl .. this might require some further investigation on what the implications are. e.g. meeting their advertising clause: https://github.com/briansmith/ring/blob/main/LICENSE#L66

Reply:
NOTE: all `N/A` licenses in that license dump are child-crates owned by the aries-vcx codebase/workspace. I'm not sure why they aren't inheriting the Apache 2.0 license in the root of the aries-vcx repo.